docs/source/conf.py

```diff
@@ -39,10 +39,18 @@
     "sphinx.ext.viewcode",
     "sphinx.ext.todo",
     "sphinx.ext.napoleon",
+    "sphinx.ext.autosectionlabel",
+    "sphinx_copybutton",
     "alabaster",
     "sitemap",
 ]
 
+#Remove '$' from copied code when using the copybutton
+copybutton_prompt_text = "$ "
```

Comment:
is it possible to also manage the `>>> ` ? (for the python console for instance)